server2/Arcadia.Assistant/Arcadia.Assistant.Calendar.Abstractions/CalendarEvent.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Arcadia.Assistant.Calendar.Abstractions
+{
+    using System.Linq;
+
+    public class CalendarEvent
```

Comment:
why do you need this copypasted? I haven't had them now because actually these structures had almost nothing in common.

Reply:
Removed